portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -123,13 +123,66 @@ definition {
 	}
 
 	@description = "Verify that a Help Text can be added"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test HelpTextCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132199 HelpTextCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+            fieldPositionNumber = "0",
+            fieldType = "Single Selection",
+            formPageNumber = "0");
+
+		var fieldSingleSelectionField = FormFields.getFieldName();
+
+		Type(
+			key_rowNumber = "1",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 1");
+
+		Type(
+			key_rowNumber = "2",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 2");
+
+		FormViewBuilder.setFieldHelpText(fieldLabel = "Single Selection Help Text");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishCopyURLLink();
+
+		var publishedFormPublicURL = Form.getPublishedFormPublicURL(
+			publishedFormPublicURL = "${publishedFormPublicURL}");
+
+		Navigator.openSpecificURL(url = "${publishedFormPublicURL}");
+
+		FormFields.viewRadioLabel(
+			fieldLabel = "Single Selection",
+			fieldName = "${fieldSingleSelectionField}");
+		
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "${fieldSingleSelectionField}",
+			fieldValue = "Single Selection Help Text");
+	
+		FormFields.selectRadioOption(
+			fieldName = "${fieldSingleSelectionField}",
+			radioOption = "Option 1");
+
+        Click(
+            key_text = "Submit",
+            locator1 = "Button#ANY");
+
+        AssertElementPresent(locator1 = "Message#SUCCESS");
```

Comment:
By using `FormPortlet.submitSuccessfully()` you are basically trying to submit the form and expects a success message, which seems perfect in this scenario that you have designed.
May I ask you to use it here?